src/avt/QtVisWindow/vtk/vtkDashedXorGridMapper2D_body.C

```diff
@@ -105,8 +106,8 @@
         }
         ftmp = p->GetPoint(pts[0]);
 
-        lastX = (int)(actorPos[0] + ftmp[0]);
-        lastY = (int)(actorPos[1] - ftmp[1]);
+        lastX = (int)(actorPos[0] + ftmp[0]) / devicePixelRatio;
```

Comment:
Would you please add comment here same as you did for vtkRubberBandMapper2D_body.C regarding the need for devicePixelRatio.